packages/server/graphql/types/ToggleTeamDrawerPayload.ts

```diff
@@ -0,0 +1,24 @@
+import {GraphQLBoolean, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import makeMutationPayload from './makeMutationPayload'
+
+export const ToggleTeamDrawerSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'ToggleTeamDrawerSuccess',
+  fields: () => ({
```

Comment:
if you return a `team` object then relay will be able to merge the new team object with the old 1 & get the right values so you don't have to write an updater for that mutation on the client